samples/snippets/src/main/java/aiplatform/CreateBatchPredictionJobTextEntityExtractionSample.java

```diff
@@ -0,0 +1,89 @@
+import com.google.cloud.aiplatform.v1.JobServiceClient;
+import com.google.cloud.aiplatform.v1.JobServiceSettings;
+import com.google.cloud.aiplatform.v1.LocationName;
+import com.google.cloud.aiplatform.v1.ModelName;
+import java.io.IOException;
+
+public class CreateBatchPredictionJobTextEntityExtractionSample {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "PROJECT";
+    String location = "us-central1";
+    String displayName = "DISPLAY_NAME";
+    String modelId = "MODEL_ID";
+    String gcsSourceUri = "GCS_SOURCE_URI";
+    String gcsDestinationOutputUriPrefix = "GCS_DESTINATION_OUTPUT_URI_PREFIX";
+    createBatchPredictionJobTextEntityExtractionSample(
+        project, location, displayName, modelId, gcsSourceUri, gcsDestinationOutputUriPrefix);
+  }
+
+  static void createBatchPredictionJobTextEntityExtractionSample(
+      String project,
+      String location,
+      String displayName,
+      String modelId,
+      String gcsSourceUri,
+      String gcsDestinationOutputUriPrefix)
+      throws IOException {
+    // The AI Platform services require regional API endpoints.
+    JobServiceSettings settings =
+        JobServiceSettings.newBuilder()
+            .setEndpoint("us-central1-aiplatform.googleapis.com:443")
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (JobServiceClient client = JobServiceClient.create(settings)) {
+      String modelName = ModelName.of(project, location, modelId).toString();
+      GcsSource gcsSource = GcsSource.newBuilder().addUris(gcsSourceUri).build();
+      BatchPredictionJob.InputConfig inputConfig =
+          BatchPredictionJob.InputConfig.newBuilder()
+              .setInstancesFormat("jsonl")
+              .setGcsSource(gcsSource)
+              .build();
+      GcsDestination gcsDestination =
+          GcsDestination.newBuilder().setOutputUriPrefix(gcsDestinationOutputUriPrefix).build();
+      BatchPredictionJob.OutputConfig outputConfig =
+          BatchPredictionJob.OutputConfig.newBuilder()
+              .setPredictionsFormat("jsonl")
+              .setGcsDestination(gcsDestination)
+              .build();
+      BatchPredictionJob batchPredictionJob =
+          BatchPredictionJob.newBuilder()
+              .setDisplayName(displayName)
+              .setModel(modelName)
+              .setInputConfig(inputConfig)
+              .setOutputConfig(outputConfig)
+              .build();
+      LocationName parent = LocationName.of(project, location);
+      BatchPredictionJob response = client.createBatchPredictionJob(parent, batchPredictionJob);
```

Comment:
Same as above -- is there an `ApiException` we can catch here if this operation didn't go through?

Reply:
Done.